synapse/handlers/events.py

```diff
@@ -144,18 +145,18 @@ async def get_stream(
 
 
 class EventHandler(BaseHandler):
-    def __init__(self, hs):
+    def __init__(self, hs: "HomeServer"):
         super(EventHandler, self).__init__(hs)
         self.storage = hs.get_storage()
 
-    async def get_event(self, user, room_id, event_id):
+    async def get_event(self, user: UserID, room_id: Optional[str], event_id: str):
```

Comment:
Seems the return type was missed here.

```suggestion
    async def get_event(self, user: UserID, room_id: Optional[str], event_id: str) -> Optional[JsonDict]:
```